### ISSUE TITLE
Training Result not well

### ISSUE THREAD

**Issue by Minotaur-CN** [salihkaragoz/pose-residual-network-pytorch#8]
Dear salihkaragoz,
Thanks for you code. The results trained on your code seems not fine as your git . Something options missing?

Thanks for your reply!

Total Step: 17372 | Total Epoch: 16
17372it [05:57, 48.62it/s]           | Epoch: 15 Total: 1:59:03 | ETA: 18:15:13 | loss:0.00240877992474
------------Evaulation Started------------
loading annotations into memory...
Done (t=0.17s)
creating index...
index created!
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2693/2693 [01:22<00:00, 32.71it/s]
Loading and preparing results...
DONE (t=0.11s)
creating index...
index created!
Running per image evaluation...
Evaluate annotation type *keypoints*
DONE (t=4.03s).
Accumulating evaluation results...
DONE (t=0.05s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] = 0.852
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets= 20 ] = 0.967
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets= 20 ] = 0.884
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets= 20 ] = 0.835
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets= 20 ] = 0.889
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] = 0.886

**Comment by salihkaragoz** [comment]
Hello, 
We have updated the repo, please clone the last version. It will give 0.892 AP at the approximately 12th epoch. There is a pre-trained model you can download and test it. 
Feel free for further questions otherwise please close this issue.
Thanks for your concern.
Regards...
